server/src/main/java/io/druid/curator/CuratorConfig.java

```diff
@@ -66,4 +69,14 @@ public void setEnableCompression(Boolean enableCompression)
   {
     this.enableCompression = enableCompression;
   }
+
+  public boolean isEnableAcl()
```

Comment:
@drcrallen  i can't seem to change this to isEnableAcl() without unit tests failing